siac/walletcmd.go

```diff
@@ -193,7 +194,10 @@ func walletinitcmd() {
 
 // walletload033xcmd loads a v0.3.3.x wallet into the current wallet.
 func walletload033xcmd(source string) {
-	password, err := speakeasy.Ask("Wallet password: ")
+	fmt.Println("Loading 033x...")
+	time.Sleep(time.Second * 3)
+	fmt.Println("Load successful.  Encrypting with current encryption password")
+	password, err := speakeasy.Ask("Need Password: ")
```

Comment:
Idea was to make it more clear that the password needed was the encryption password to the current wallet.